scapy/asn1fields.py

```diff
@@ -441,10 +435,7 @@ def __init__(self, name, default, *args, **kwargs):
             raise ASN1_Error(err_msg)
         self.implicit_tag = None
         for kwarg in ["context", "explicit_tag"]:
-            if kwarg in kwargs:
-                setattr(self, kwarg, kwargs[kwarg])
-            else:
-                setattr(self, kwarg, None)
+            setattr(self, kwarg, kwargs.get(kwarg))
```

Comment:
Shouldn’t it be
`kwargs.get(kwarg, None)`
?

Reply:
kwargs.get(kwarg) is exactly the same as kwargs.get(kwarg, None)

default is None

But if you think that strict "explicit is better than implicit" is more valuable/accurate
I am quite open for agreement with you in this matter :)

Reply:
You are right. I thought it would have triggered an undefined error, but I was wrong

Reply:
@gpotter2 that's the case for `.pop()`... which is a bit weird as you might think both would have the same API.

Reply:
Exactly:-)